package.json

```diff
@@ -65,7 +65,7 @@
     "lint-staged": "4.0.1",
     "meetup-web-components": "3.2.1592",
     "meetup-web-mocks": "1.0.217",
-    "mwp-cli": "3.0.327",
+    "memwatch-next": "0.3.0",
```

Comment:
this is just moving a devDependency from mwp-consumer into the repo root, as recommended by Lerna (our monorepo management tool)